### ISSUE TITLE
UI Improvements

### ISSUE THREAD

**Issue by ameya-sirpotdar** [ameya-sirpotdar/options#71]
I think we are occupying entire row for each one of them:
- Tickers: It should be dropdown with check boxes for the following tickets by default. QQQ, SPY, MSFT, AAPL, GOOG, META, NFLX, TSLA, NVDA, AMZN
- Delta: Default 0.30
- Exp Date
- VIX

Make the working area wider as well.
Make sure the current functionality is not broken.

**Comment by ameya-sirpotdar** [comment]
Resolved in PR #72.
